src/components/search/curated/index.tsx

```diff
@@ -1,11 +1,13 @@
 import SearchJavaScript from './javascript'
 import SearchReact from './react'
 import SearchGraphql from './graphql'
+import SearchTypeScript from './typescript'
```

Comment:
@VontaJamal we changed how we added curated pages so I updated it here

Reply:
understood